index.js

```diff
@@ -8,127 +8,192 @@ module.exports = {
+        'arrow-spacing': ['error', { before: true, after: true }],
+        'prefer-arrow-callback': 'error',
+
+        // other
+        'new-cap': ['error', {
+            capIsNewExceptions: ['Polymer', 'Router'],
+        }],
+        'linebreak-style': ['error', 'unix'],
+        'indent': ['error', 4, { SwitchCase: 1 }],
+        'quotes': ['error', 'single', 'avoid-escape'],
 
         // spacing
-        'space-in-parens': [ 0, 'always' ],
-        'computed-property-spacing': [ 0, 'always' ],
-        'space-before-function-paren': [ 0, 'always' ],
-        'array-bracket-spacing': [ 0, 'always' ],
-        'object-curly-spacing': [ 0, 'always' ],
+        'space-in-parens': ['error', 'never'],
+        'computed-property-spacing': ['error', 'never'],
+        'space-before-function-paren': ['error', {
+            anonymous: 'always',
+            asyncArrow: 'always',
+            named: 'never',
+        }],
+        'array-bracket-spacing': ['error', 'never'],
+        'object-curly-spacing': ['error', 'always'],
 
-        'operator-linebreak': [ 2, 'before' ],
+        'operator-linebreak': ['error', 'before'],
 
-        // this used to be [2, 5], but when updating to ESLint 5,
+        // this used to be ['error', 5], but when updating to ESLint 5,
         // complexity measurement got better and so the existing codebase
         // failed to lint
-        'complexity': [ 2, 12 ],
+        'complexity': ['error', 12],
 
-        'no-unused-expressions': [ 2, {
-            'allowTernary': true,
-            'allowShortCircuit': true
-        } ],
+        'no-unused-expressions': ['error', {
+            allowTernary: true,
+            allowShortCircuit: true,
+        }],
 
-        'no-unused-vars': [ 2, {
-            'argsIgnorePattern': '^next$'
+        'no-unused-vars': ['error', {
+            argsIgnorePattern: '^next$',
         }],
 
         // negated conditions are more difficult to understand. Code can be
         // made more readable by inverting the condition instead.
-        'no-negated-condition': [ 2 ],
-
-        'no-multi-spaces': ["error", {
-            "ignoreEOLComments": true,
+        'no-negated-condition': 'error',
 
+        'no-multi-spaces': ['error', {
+            ignoreEOLComments: true,
         }],
 
-        'max-len': [ 2, 100, 4, {
-            ignoreUrls: true
+        'max-len': ['error', {
+            code: 100,
+            tabWidth: 4,
+            ignoreUrls: true,
         }],
 
         // disallow weak equal signs (= instead of ==)
-        'eqeqeq': ["error", "always", { "null": "ignore" }],
+        'eqeqeq': ['error', 'always', { null: 'ignore' }],
 
         // jsdoc is required but doesn't have to be valid
         // this is in order to enforce some comments on classes and method
         // without requiring a lot of redundant boilerplate
-        'require-jsdoc': [ 2 ],
-        'valid-jsdoc': [ 0 ],
+        'require-jsdoc': 'error',
+        'valid-jsdoc': 'off',
```

Comment:
This was deprecated in v5.10.0 so why is this here?

See https://eslint.org/docs/latest/rules/valid-jsdoc